src/javascripts/modules/ExpandListItem.js

```diff
@@ -0,0 +1,26 @@
+import { debounce } from 'utility/debounce'
+
+export default class ExpandListItem {
+  constructor(el) {
+    this.el = el
+    this.tag = el.parentNode
+    this.el.addEventListener('click', this.expand)
+
+    let resizeRate = debounce(this.resize, 250)
+    window.addEventListener('resize', resizeRate)
```

Comment:
i'd break this constructor method into a couple of different methods just to keep things clear. it looks like in your constructor you a) define the parent element b) define some variables c) define some event listeners.

in that case, i'd rewrite this constructor as:
```js
constructor(el) {
  this.el = el

  this.defineVars() // add any variables
  this.listen() // add any event listeners
}
```

this way you can break the appropriate portions of this into well-named methods that's easy for folks to understand and add to in the future. on top of that it keeps your constructor clean and focused.

Reply:
+1. tho fwiw (and this is more an interesting note for @benjtinsley) wrt `this.defineVars()`: Nate's told me he prefers settings variables in the constructor, unlike our unofficial team standard of having a variable-setting function

Reply:
@olets i can appreciate both ways. not a real stickler for creating a separate `defineVars` function if the constructor looks consistent and clean

Reply:
I prefer Nate's approach of setting/storing variables in the constructor, especially when there's only a few variables to set, but both approaches are fine and clear.

Reply:
You can combine your debounce lines into one, like this:
```js
window.addEventListener('resize', debounce(this.resize, 250))
```
Not sure if you'd need the fat arrow in there or not:
```js
window.addEventListener('resize', debounce(() => this.resize(), 200))
```

Also, your resize method isn't technically doing any resizing, it's "resetting" any tags that are hidden and making them visible for mobile. So I'd suggest renaming the method to indicate what action is being taken, something like `setVisibility()`. One other thing, do you need to reset the visibility when going back from mobile to desktop? If so, you'd need another matchMedia expression using min-width to make them hidden when going to desktop.